src/GLideNUI/GLideNUI.cpp

```diff
@@ -29,27 +29,34 @@ int openConfigDialog(const wchar_t * _strFileName, const char * _romName, bool &
 	if (config.generalEmulation.enableCustomSettings != 0 && _romName != nullptr && strlen(_romName) != 0)
 		loadCustomRomSettings(strIniFileName, _romName);
 
-#ifndef M64P_GLIDENUI
-	int argc = 0;
-	char * argv = 0;
-	QApplication a(argc, &argv);
+	QCoreApplication* pApp = QCoreApplication::instance();
+	bool hasApp = pApp != nullptr;
 
-	QTranslator translator;
-	if (translator.load(getTranslationFile(), strIniFileName))
-		a.installTranslator(&translator);
-#endif // M64P_GLIDENUI
+	if (!hasApp) {
+		int argc = 0;
+		char * argv = 0;
+		pApp = new QApplication(argc, &argv);
+
+		QTranslator translator;
+		if (translator.load(getTranslationFile(), strIniFileName))
+			pApp->installTranslator(&translator);
```

Comment:
If hasApp is true, translation will not be applied, will it?

Reply:
that's correct